packages/api/migrations/20211015015859-create-domain.js

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize, DataTypes) => {
+    await queryInterface.createTable('domains', {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4,
+      },
+      name: {
+        type: Sequelize.STRING,
+        allowNull: false,
+      },
+      createdAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+      updatedAt: {
+        allowNull: false,
+        type: Sequelize.DATE,
+      },
+    });
+    await queryInterface.bulkInsert('domains', [
+      { 
+        id: 'bc3cd395-226c-46be-9b80-89c5f1c944db', 
```

Comment:
If this works that's fine and I don't think you necessarily need to change it, but typically when inserting seed data like this you omit the ID and let the database create it for you.

Reply:
For some reason, it was giving us an error when we didn't include an ID. It might have been the way we were doing it but adding an ID was the way we were able to get it working.